### DISCUSSION TITLE
[RELEASE] Sniper Range

### DISCUSSION THREAD

**Discussion by Bobthebob5** [nzp-team/nzportable#755] [Map Releases]
A fun map idea I wanted to execute. One room and zombies coming. Use the snipers and aim for the head! 

![SniperRange](https://github.com/nzp-team/nzportable/assets/167271269/768a3db4-90ed-4405-84df-6f7390be331b)

#[DOWNLOAD](https://github.com/user-attachments/files/15749908/Sniperange.zip)
 

Tested on PC and PSP 3000

**Comment by Official-KEX** [comment]
> [!TIP]
> Give a german man a Kar98 and he's happy. 😄 
---
> [!NOTE]
> Im a happy man now. ❤️

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org
